src/qililab/qprogram/qblox_compiler.py

```diff
@@ -396,7 +396,7 @@ def _handle_set_offset(self, element: SetOffset):
             else convert(element.offset_path0)
         )
         offset_1 = (
-            self._buses[element.bus].variable_to_register[element.offset_path1]
+            self._buses[element.bus].variable_to_register[element.offset_path1]  # type: ignore[index]
             if isinstance(element.offset_path1, Variable)
             else convert(element.offset_path1)
         )
```

Comment:
For Qblox we should raise an error if offset_1 is `None`, since Q1ASM requires both values to be given.

Reply:
thanks for the observation I implemented the changes